### ISSUE TITLE
putty and winscp use weak key exchange

### ISSUE THREAD

**Issue by jumanjiman** [jumanjihouse/wormhole#51]
## tl;dr

putty and winscp no longer connect to wormhole devenvs

workarounds: 

* use chrome browser and the [ssh extension](https://chrome.google.com/webstore/detail/secure-shell/pnhechapfaindjhompbnflcldabbghjo) (add bookmark to the extension for quick access)

* use cygwin or mingw

## more info

commit 83c1f63fbf2ecf6f9b380d8ca3f53e0429748049 improved security of wormhole sshd based on tips and description at https://stribika.github.io/2015/01/04/secure-secure-shell.html

notably, we no longer allow diffie-hellman-group1 or diffie-hellman-group14, and these are the _only_ key exchange algos supported by putty according to http://the.earth.li/~sgtatham/putty/0.63/htmldoc/Chapter4.html#config-ssh-kex (the docs link at http://www.chiark.greenend.org.uk/~sgtatham/putty/docs.html)

**Comment by jumanjiman** [comment]
there are other ssh-related discussions at https://github.com/stribika/stribika.github.io/issues

**Comment by jumanjiman** [comment]
to do: update readme with suggestion

**Comment by planet** [comment]
@mazad @dchin so you are aware

**Comment by jumanjiman** [comment]
from https://github.com/stribika/stribika.github.io/issues/15#issuecomment-69597212

> diffie-hellman-group-exchange-sha256 is available (or will be soon) in the latest PuTTY: http://tartarus.org/~simon-git/gitweb/?p=putty.git;a=blob_plain;f=sshdh.c;hb=HEAD

**Comment by jumanjiman** [comment]
https://github.com/stribika/stribika.github.io/wiki/Secure-Secure-Shell

**Comment by planet** [comment]
https://github.com/ISEexchange/chocopackages/commit/94aa106bd6003cafa611d1820f5cf3245897e508 

resolved by updating the chocopackage for "PuttyTray" to latest

https://github.com/ISEexchange/chocopackages/commit/94aa106bd6003cafa611d1820f5cf3245897e508

http://chocolatey.ise.com/packages/putty/0.63.22

**Comment by jumanjiman** [comment]
:+1:
